ClassTypes/PowerShell/EthicalHacking/Create-EthicalHackingLabplanAndLab.ps1

```diff
@@ -1,19 +1,32 @@
+<#
+The MIT License (MIT)
+Copyright (c) Microsoft Corporation  
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.  
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE. 
+.SYNOPSIS
+    Creates a Azure Lab Services lab plan and lab that can be used to create an ethical hacking lab.
+.DESCRIPTION
+    Creates a Azure Lab Services lab plan and lab that can be used to create an ethical hacking lab.
+.PARAMETER Username
+    The username for the local administrator account on the VM.
+.PARAMETER Username
+    The password for the local administrator account on the VM.   See https://learn.microsoft.com/azure/virtual-machines/windows/faq#what-are-the-password-requirements-when-creating-a-vm-  
+.PARAMETER Location
+    Location name for Azure region where the lab plan and lab should reside.  Run `Get-AzLocation | Format-Table` to see all available.
+#>
+
 [CmdletBinding()]
 param(
-    [parameter(Mandatory = $false)]
-    [string]$Email,
-
-    [parameter(Mandatory = $false, HelpMessage = "Default username for all VMs")]
-    [string]$Username = "AdminUser",
 
-    [parameter(Mandatory = $false, HelpMessage = "Default password for all VMs")]
-    [string]$Password = "P@ssword1!",
+    [parameter(Mandatory = $true, HelpMessage = "Username for all VMs")]
+    [string]$UserName = "AdminUser",
 
-    [parameter(Mandatory = $false, HelpMessage = "Default location for lab plan")]
-    [string]$Location = "centralus",
+    [parameter(Mandatory = $true, HelpMessage = "Password for all VMs")]
```

Comment:
Should we validate the password?  I'll send you a direct message on this...

Reply:
I added a check for the regex you sent me.  16 characters is pretty long, so I think it may be a common issue.